scenes/game/local_game.gd

```diff
@@ -2765,6 +2765,17 @@ func is_effect_condition_met(performing_player : Player, effect, local_condition
 			return not performing_player.strike_stat_boosts.critical
 		elif condition == "choose_cards_from_top_deck_action":
 			return decision_info.action == effect["condition_details"]
+		elif condition == "total_powerup_greater_or_equal":
+			var amount = effect["condition_amount"]
+			var total_powerup = performing_player.strike_stat_boosts.power
+			return total_powerup >= amount
+		elif condition == "opponent_total_guard_greater_or_equal":
+			var amount = effect["condition_amount"]
+			var card = active_strike.get_player_card(other_player)
```

Comment:
I don't understand why you can't use the get_total_guard function here.

Reply:
if it isn't working, we need to debug and fix it as guard is probably broken somehow